code/addons/links/package.json

```diff
@@ -21,9 +21,37 @@
     "url": "https://opencollective.com/storybook"
   },
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./manager": {
+      "require": "./dist/manager.js",
+      "import": "./dist/manager.mjs",
+      "types": "./dist/manager.d.ts"
+    },
+    "./preview": {
+      "require": "./dist/preview.js",
+      "import": "./dist/preview.mjs",
+      "types": "./dist/preview.d.ts"
+    },
+    "./react": {
+      "require": "./dist/react/index.js",
+      "import": "./dist/react/index.mjs",
+      "types": "./dist/react.d.ts"
```

Comment:
While testing in `examples/official-storybook`, I noticed that there are imports from `@addon-links/react`, so I needed to add this endpoint here. 

Please, let me know if there's a better approach!

<img width="469" alt="Screenshot 2022-08-09 at 17 34 29" src="https://user-images.githubusercontent.com/57359698/183695199-61f46405-fadd-4c56-8c83-b5125efd7dda.png">

Reply:
I think we'll want to deprecate this at some point in the future, but that will be a separate issue/PR, so great work!